### PR TITLE
test/system: podman run update /etc/hosts

### PR DESCRIPTION
**Pull request by chuanchang**
This test case is used for covering rhbz#1902979.

Signed-off-by: Alex Jia <ajia@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@edsantiago PTAL

File: test/system/030-run.bats [hunk 1]
```diff
@@ -736,4 +736,11 @@ EOF
     is "$output" "$random_1" "output matches STDIN"
 }
 
+# rhbz#1902979 : podman run fails to update /etc/hosts when --uidmap is provided
+@test "podman run update /etc/hosts" {
+    HOSTS=$(random_string 25)
```
Comment (edsantiago) [inline test/system/030-run.bats:741]: Suggestion: use singular (`HOST`), not plural. Plural is confusing here.

File: test/system/030-run.bats [hunk 2]
```diff
@@ -736,4 +736,11 @@ EOF
...
+    run_podman run --uidmap 0:10001:10002 --rm --hostname $HOSTS $IMAGE grep $HOSTS /etc/hosts
+    is "${lines[0]}" ".*$HOSTS.*"
```
Comment (edsantiago) [inline test/system/030-run.bats:743]: This check is unnecessary: if the `grep` above were to fail, `run_podman` would issue a failure.

File: test/system/030-run.bats [hunk 3]
```diff
@@ -736,4 +736,11 @@ EOF
     is "$output" "$random_1" "output matches STDIN"
 }
 
+# rhbz#1902979 : podman run fails to update /etc/hosts when --uidmap is provided
+@test "podman run update /etc/hosts" {
```
Comment (edsantiago) [inline test/system/030-run.bats:740]: Please make this test name more descriptive, such as by including something about uidmap.

**Review by edsantiago — CHANGES_REQUESTED** [review]
Thank you for your PRs!

A few fixes needed. And, in this and all your other PRs, could you resubmit with a more descriptive commit message? A commit message should be something that a future maintainer can read without reference to other sources. It's OK to link to external BZs or issues, but please give a short description of what the problem actually is. (This is especially important with BZs that have a lot of RH-internal-only comments).

**Comment by umohnani8** [comment]
@chuanchang please rebase

File: test/system/030-run.bats (hunk 3 above)
Comment (rhatdan) [reply test/system/030-run.bats]: Fixed

**Pull request closed without merging**